High level installation instructions

leftclickben opened the pull request:
I wrote some instructions for installing everything.  There wasn't an ideal place to put them so I've put them here in the `shop-theme` repo, which had a more-or-less empty `README.md` file.

Commit:
WIP High-level installation instructions (magento+data+theme)

Commit:
Finish writing instructions for deployment of theme and database.

Commit:
Add missing step of moving media files to file system.

Commit:
Line wrapping.

Commit:
Make file system tree more readable.

leftclickben commented:
I've set this to merge into `rwahs/RebaseOntoLuma`, but after it is merged, we should merge that into `rwahs/master` because that is the direction we have chosen to go in.

leftclickben commented:
These instructions were based on what I did to get it working in both staging and UAT, but have not been independently tested, so they might not be perfect.  I don't know if it's worth spinning up a micro instance just to test the instructions?

kehh commented on README.md:
```diff
@@ -1,2 +1,123 @@
+
+* Version: 2.0.x
+* Method: composer
+
```
You also need an access key as defined at http://devdocs.magento.com/guides/v2.0/install-gde/prereq/connect-auth.html

Commit:
Add requirement for access key.

kehh commented on README.md:
````diff
@@ -1,2 +1,127 @@
+installing the database seed in the next step.
+
+```
+cd /srv/sites/documentation-shop.histwest.org.au
````
set permissions on bin/magento (and other bits as well)

Commit:
Be more explicit about file system ownership/permissions.

+ Take the note out of `References` section.
+ Add an explicit step to set the permissions and ownership.
+ Put prerequisites into a separate section, and replace with
  a link to the Magento2 documentation.

Commit:
Formatting

kehh commented:
so anyway:

``` bash
cp -r ../website-theme app/design/frontend/Gaia/rwahsluma
```

Didn't seem to work.

kehh commented:
and then I kicked myself many times when I realised my mistake!!!

kehh commented:
![w00t](https://cloud.githubusercontent.com/assets/12571/17876994/f01477f8-6911-11e6-859a-f6172dd7a5bf.png)
